tests/FSharp.Test.Utilities/TestFramework.fs

```diff
@@ -350,7 +357,7 @@ let config configurationName envVars =
     let FSC = requireArtifact ("fsc" ++ configurationName ++ fscArchitecture ++ "fsc.dll")
 #endif
     let FSCOREDLLPATH = requireArtifact ("FSharp.Core" ++ configurationName ++ fsharpCoreArchitecture ++ "FSharp.Core.dll")
-
+    let DOTNETFSCCOMPILERPATH = requireArtifact ("fsc" ++ configurationName ++ dotnetArchitecture ++ "fsc.dll")
```

Comment:
Is it the same as the FSC path above?

Reply:
only when building coreclr on desktop framework build fsc points to fsc.exe